docs/_docs/01_getting-started/terminology.md

```diff
@@ -0,0 +1,209 @@
+
+Note that units don't technically need to call their configuration files `terragrunt.hcl` (that's configurable via the [--terragrunt-config](/docs/reference/cli-options/#terragrunt-config), soon to be rebranded [--config](https://github.com/gruntwork-io/terragrunt/issues/3445)), and users don't technically need to use a root `terragrunt.hcl` file or to name it that. This is the most common pattern followed by the community, however, and deviation from this pattern should be justified in the context of the project. It can help others with Terragrunt experience understand the project more easily if industry standard patterns are followed.
+
+While not a requirement, a general tendency experienced when working with Terragrunt is that units tend to decrease in size. This is because Terragrunt makes it easy to segment pieces of infrastructure into their own state, and to have them interact with each other through the use of [dependency blocks](/docs/reference/config-blocks-and-attributes#dependency). Smaller units are quicker to update, easier to reason about and safer to work with.
+
+Units typically represent a minimal useful piece of infrastructure that should be independently managed.
+
+e.g. A unit might represent a single VPC, a single database, or a single server.
+
+### Stack
+
+A stack is a collection of units managed by Terragrunt. There is ([as of writing](https://github.com/gruntwork-io/terragrunt/issues/3313) work underway to provide a top level artifact for interacting with stacks via a `terragrunt.stack.hcl` file, but as of now, stacks are generally defined by a directory with a tree of units. Units within a stack can be dependent on each other, and can be updated in a specific order to ensure that dependencies are resolved in the correct order.
+
+The design of `terragrunt.stack.hcl` files is to ensure that they function entirely as a convenient shorthand for an equivalent directory structure of units. This is to ensure that users are able to easily transition between the two paradigms, and are able to decide for themselves which approach to structuring infrastructure is most appropriate for their use case.
+
+Stacks typically represent a collection of units that need to be managed in concert.
+
+e.g. A stack might represent a collection of units that together form a single application environment, a business unit, or a region.
+
+### Module
+
+A module is a collection of OpenTofu/Terraform configurations defined with files ending in `.tf` (or `.tofu` in the case of OpenTofu) that represent a general pattern of infrastructure that can be instantiated multiple times.
```

Comment:
The `module` section seemed vague to me - maybe start with defining them as an existing opentofu/terraform construct and link to https://opentofu.org/docs/language/modules/ - then add how they integrate into Terragrunt. E.g. should I import a module into my terragrunt unit?